examples/state_nontrivial/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "state_nontrivial"
+version = "0.1.0"
+authors = ["Stephan Buys <stephan.buys@panoptix.co.za>"]
```

Comment:
No authors in examples, and version should be `0.0.1`. This should be in the workspace via `workspace = "../.."`.